f600233d1b154d507b782f84604bcb12/update/sys_ws_operation_1946bd8edb64a918b1227ea5f396197f.xml

```diff
@@ -0,0 +1,115 @@
+	var error = new sn_ws_err.ServiceError();
+
+	// Ensure TO not already in use
+	var toRecord = new GlideRecord("x_g_dis_atat_task_order");
+	toRecord.addQuery("task_order_number", request.queryParams.taskOrderNumber);
+	toRecord.query();
+	if (toRecord.next()) {
+		var message = "The given Task Order is already in use in a Portfolio";
+		if (toRecord.portfolio) {
+			message += " named " + toRecord.portfolio.name;			
+		}
+		return {
+			code: "0001",
+			message: message
+		};
+	}
+	
+	var acqPackage = new GlideRecord("x_g_dis_atat_acquisition_package");
+	acqPackage.get(request.queryParams.acqusitionPackageSysId);
+	var eda = new Eda();
+
+	// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+	// CODE BELOW for testing only - remove when JWCC TOs are available in EDA GT environment
+	// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+	var dummyRecord = new GlideRecord("x_g_dis_atat_dapps_dummy_data");
+	if (request.queryParams.taskOrderNumber == "1000000000000") {
+		dummyRecord.addQuery("name", "AWS");
+		dummyRecord.query();
+		dummyRecord.next();
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, dummyRecord.content);
+	} 
+	else if (request.queryParams.taskOrderNumber == "2000000000000") {
+		dummyRecord.addQuery("name", "Azure");
+		dummyRecord.query();
+		dummyRecord.next();
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, dummyRecord.content);
+	} 	
+	else if (request.queryParams.taskOrderNumber == "3000000000000") {
+		dummyRecord.addQuery("name", "GCP");
+		dummyRecord.query();
+		dummyRecord.next();
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, dummyRecord.content);
+	} 	
+	else if (request.queryParams.taskOrderNumber == "4000000000000") {
+		dummyRecord.addQuery("name", "Oracle");
+		dummyRecord.query();
+		dummyRecord.next();
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, dummyRecord.content);
+	} 	
+	else if (request.queryParams.taskOrderNumber == "5000000000000") {
+		dummyRecord.addQuery("name", "Non-JWCC");
+		dummyRecord.query();
+		dummyRecord.next();
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, dummyRecord.content);
+	} 		
+	// END EDA HARD-CODING
+	
+	// Make GET request to endpoint
+	var my_procurement_doc = eda.pdsRequest(request.queryParams.taskOrderNumber);
+	if (my_procurement_doc.error){
+		if (my_procurement_doc.status_code == "204") {
+			return {
+				code: "0002",
+				message: "Could not find given Task Order."
+			};
+		}
+		else {		
+			var pdsReqError = new sn_ws_err.ServiceError();
+			pdsReqError.setMessage(my_procurement_doc.error);
+			pdsReqError.setDetail("Check System>System Logs>Errors for thrown error from edaPdsRequest");
+			return pdsReqError;			
+		}
+
+	}
+	else{
+		return eda.extractPdsSummary(request.queryParams.taskOrderNumber, my_procurement_doc.body);
+		//return my_procurement_doc.body;
```

Comment:
Can this comment be removed? It seems like it might have been left from a previous implementation.

Reply:
Thanks. I may actually add in another test endpoint that does this so we can debug responses, but for now we should remove that.